test/scripts/upgrade-ovn.sh

```diff
@@ -6,6 +6,13 @@ set -ex
 export KUBECONFIG=${KUBECONFIG:-${HOME}/ovn.conf}
 export OVN_IMAGE=${OVN_IMAGE:-ovn-daemonset-fedora:pr}
 
+set -ex
+ARCH=""
+case $(uname -m) in
+    x86_64)  ARCH="amd64" ;;
+    aarch64) ARCH="arm64"   ;;
+esac
+
```

Comment:
I don't know if it's worth it but we can use bash hash tables here.
```bash
declare -A archs=( ["x86_64"]="amd64" ["aarch64"]="arm64")
ARCH="${archs[$(uname -m)]}"
```

Reply:
Not a must you can skip this

Reply:
just a different way to do the same thing? Or maybe I am missing a benefit?